src/tasks/DockerTaskProvider.ts

```diff
@@ -49,6 +50,8 @@ export abstract class DockerTaskProvider implements TaskProvider {
                 context.actionContext.telemetry.properties.dockerPlatform = context.platform;
                 context.actionContext.telemetry.properties.orchestration = 'single' as DockerOrchestration; // TODO: docker-compose, when support is added
                 await this.executeTaskInternal(context, task);
+
+                dockerTaskEndEventListener.fire({name: task.name, success: true});

```

Comment:
It may be OK but I want call out--this means the event listener is shared between `docker-build` tasks and `docker-run` tasks. Does that cause any issues? (I can't think of any off the top of my head)

Reply:
Well, it runs pretty normal, I didn't see any issues in my testing so far. It's a good point, let's keep an eye out for CTI's testing but from my perspective it looks fine.